src/App.js

```diff
@@ -1,16 +1,44 @@
 import React from "react";
 import logo from "./logo.png";
 import "./App.css";
+import Clock from "./components/Clock";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
```

Comment:
you should try to let vscode intellisense handle imports for you. then it can include all the classes from the same source. see my video: 

https://user-images.githubusercontent.com/40119924/191027477-5db6dcff-badf-4438-92c8-4ea42e08ae3f.mov

Reply:
![image](https://user-images.githubusercontent.com/30802551/191047993-cc2e1bfe-a790-4295-a3fe-b52625e5925e.png)
now it has the weird .../esm/Row & .../esm/Col

Reply:
hmm, i'm not sure what's causing the issue. since imports are still working, and this is really just a cleanliness issue, let's ignore it for now. we'll revisit it again if we can find out why the imports are pointing to subfolders.